src/applications/edu-benefits/components/EducationWizard.jsx

```diff
@@ -362,60 +363,131 @@ export class EducationWizard extends React.Component {
+                            })
+                          }
+                        >
+                          See approved STEM programs
+                        </a>
+                      </li>
+                    </ul>
+                  </div>
+                ) : (
+                  <div>
+                    <strong>
+                      To be eligible for the{' '}
+                      <a
+                        href="https://benefits.va.gov/gibill/fgib/stem.asp"
+                        onClick={() =>
+                          recordEvent({
+                            event: 'edu-navigation',
+                            'edu-action': 'stem-scholarship',
+                          })
+                        }
+                      >
+                        Edith Nourse Rogers STEM Scholarship
+                      </a>
+                      , you must meet all the requirements below.
+                    </strong>
+                    <ul>
+                      <li className="ul-styling">
+                        <b>Education benefit:</b> You're using or recently used
+                        Post-9/11 GI Bill or Fry Scholarship benefits.
+                      </li>
+                      <li>
+                        <b>STEM degree:</b>
+                        <ul className="circle-bullet ul-styling li-styling2">
+                          <li className="li-styling">
+                            You're enrolled in a bachelor’s degree program for
+                            science, technology, engineering, or math (STEM),{' '}
+                            <b>or</b>
+                          </li>{' '}
+                          <li>
+                            {' '}
+                            You've already earned a STEM bachelor’s degree and
+                            are pursuing a teaching certification.{' '}
+                            <a
+                              href="https://benefits.va.gov/gibill/docs/fgib/STEM_Program_List.pdf"
+                              onClick={() =>
+                                recordEvent({
+                                  event: 'edu-navigation',
+                                  'edu-action': 'see-approved-stem-programs',
+                                })
+                              }
+                            >
+                              See eligible degree programs
+                            </a>
+                          </li>
+                        </ul>
+                      </li>
+                      <li className="ul-styling">
+                        <b>Remaining entitlement:</b> You've used all of your
+                        education benefits or are within 6 months of using all
+                        your benefits when you submit your application.{' '}
+                        <a
+                          className="checkBenefitsLink"
+                          href="../gi-bill/post-9-11/ch-33-benefit/"
```

Comment:
I don't think we use relative links throughout the site. I think we usually link from the root of the site like -

```suggestion
                          href="/education/gi-bill/post-9-11/ch-33-benefit/"
```